src/main.hpp

```diff
@@ -33,4 +34,6 @@ extern int g_nNewNice;
 
 extern int g_nXWaylandCount;
 
+extern pthread_t g_mainThread;
```

Comment:
This is never initialized it seems?

I'd prefer to use `g_bRun` instead (followed by `nudge_steamcompmgr` to wake up the steamcompmgr thread).

Reply:
g_mainThread is already in `main.cpp`, and is used to kill the thread in `steamCompMgrThreadRun`. I'm just making it external. Good ideal re. using `g_bRun` though, I was looking for some equivalent of this, much cleaner and less aggressive.

Reply:
Adjusted to use g_bRun, tested and works just as well. Thanks for the advice.